lib/settings.rb

```diff
@@ -26,8 +27,12 @@ def initialize config_file_path
       @config = YAML.load_file(config_file_path)
 
       if @config
-        @developer_public_key = @config["developer_public_key"]
-        @member_token = @config["member_token"]
+        @developer_public_key   = @config["developer_public_key"]
+        @member_token           = @config["member_token"]
+        @not_done_columns       = @config["not_done_columns"].freeze
```

Comment:
I would suggest to do

```ruby
        @not_done_columns       = @config["not_done_columns"].freeze || ["Sprint Backlog", "Doing"]
```

here, so we have the old behavior by default. Then it also works with our data out of the box.